src/sonic-yang-models/yang-models/sonic-storm-control.yang

```diff
@@ -0,0 +1,56 @@
+module sonic-storm-control {
+    namespace "http://github.com/Azure/sonic-storm-control";
+    yang-version "1";
+
+    prefix ssc;
+
+    import sonic-port {
+        prefix prt;
+    }
+
+    organization 
+        "SONiC";
+
+    contact 
+        "SONiC";
+
+    description
+        "This module defines configuration and operational state
+        data for broadcast, unknown-unicast and unknown-multicast (BUM) storm-control.  
+        storm-control can be enabled on ethernet interfaces. 
+        Rate is defined in bits-per-second. All three types of storm-control can be 
+        enabled on an interface.";
+
+    revision 2019-09-03 {
+        description 
+            "Initial revision";
+    }
+
+    container sonic-storm-control {
+        container PORT_STORM_CONTROL {
+            list PORT_STORM_CONTROL_LIST {
+                key "ifname storm_type";
+                /*scommon:key-pattern "PORT_STORM_CONTROL|{ifname}:{storm_type}";*/
```

Comment:
Why do we have ":" in the key pattern, we should use "|"

Reply:
Updated